internal/cli/kraft/cloud/compose/start/start.go

```diff
@@ -117,7 +118,12 @@ func (opts *StartOptions) Run(ctx context.Context, args []string) error {
 			return fmt.Errorf("service '%s' not found", serviceName)
 		}
 
-		instances = append(instances, service.Name)
+		name := strings.ReplaceAll(fmt.Sprintf("%s-%s", opts.Project.Name, service.Name), "_", "-")
+		if cname := service.ContainerName; len(cname) > 0 {
```

Comment:
Why is this condition for? Is there a `containername` field name in the Compose specification?

Reply:
yup